Makefile

```diff
@@ -126,7 +126,7 @@ test-go: test-go-unit test-go-integration
 .PHONY: test-go-unit
 test-go-unit: ## Run unit tests for backend with flags.
 	@echo "test backend unit tests"
-	$(GO) test -short -covermode=atomic -timeout=30m ./pkg/... $(GO_DECOUPLED_CORE_PLUGIN_TESTS)
+	$(GO) test -tags requires_buildifer -short -covermode=atomic -timeout=30m ./pkg/... $(GO_DECOUPLED_CORE_PLUGIN_TESTS)
```

Comment:
Typo:

```suggestion
	$(GO) test -tags requires_buildifier -short -covermode=atomic -timeout=30m ./pkg/... $(GO_DECOUPLED_CORE_PLUGIN_TESTS)
```

However, if the typo is fixed, the `test-go-unit` target will fail when `buildifier` is not installed.

It will fail for a lot of people who run tests locally, since `buildifier` is required only when editing starlark files.

The test that requires buildifier is just this one (for now):

https://github.com/grafana/grafana/blob/290fc3aadf4400f224bd028806ad2e3975a19531/pkg/build/cmd/verifystarlark_test.go#L15

PR: #62597

I thought about two solutions:

1. We can either remove the `//go:build requires_buildifier` directive at the top of the file and skip the test when buildifier is not installed (however, this may cause the test to be skipped accidentally)

2. Or we can add a new `Makefile` target:

```Makefile
.PHONY: test-go-buildifier
test-go-buildifier:
	$(GO) test -tags requires_buildifier -short -covermode=atomic -timeout=30m ./pkg/build/cmd/...
```

And execute it in `test_backend_step` before `make test-go-unit` (only in CI):

```starlark
        "commands": [
            "make test-go-buildifier",
            "make test-go-unit",
        ],
```

I'd prefer proposal 2 as it avoids the test to be skipped accidentally, but I'd like to hear from @grafana/grafana-delivery too as they may have more solutions

Reply:
again very good catch! I copy-pasted the flag from the original file so this means that the argument has never been really used so I can just remove it, isn't it?

If we want to properly add support for what it was the original intention of the flag, it can be done as a separate issue.

Reply:
Oh! :fearful: 
Yes, the test is currently not run in CI due to the typo. You can check any recent drone run where  `test_backend` step is run and you will see `-tags requires_buildifer` (typo) instead of `-tags requires_buildifier`.

Then yes I think it requires fixing in a separate PR